### PR TITLE
RDM-9554 Data for BEFTA complex fields ordering

### PR DESCRIPTION
**Pull request by rafalkalita**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-9554

### Change description ###
Added new case type FT_ComplexFieldsOrdering

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ plugins {
 
 apply plugin: 'java'
 
-def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "5.12.0-SNAPSHOT"
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "5.13.0-RDM-9554"
```
Comment (kiran-yenigala-hmcts) [inline build.gradle:10]: we are doing the pre-release instead of changing things in build.gradle..

**Review by kiran-yenigala-hmcts — COMMENTED** [review]
minor comment otherwise looks good to me

File: build.gradle (hunk 1 above)
Comment (rafalkalita) [reply build.gradle:10]: Did the pre-release to be able to build the data-store PR. Will need to release it under a 5.13.0 version when PR is approved. Then bump in master to 5.14.0-SNAPSHOT.

**Review by kiran-yenigala-hmcts — APPROVED** [review]
LGTM

**Pull request merged**